docs/resources/ecosystem/community.md

```diff
@@ -551,6 +551,51 @@ object CircuitBreakerExample extends zio.App {
 }
 ```
 
+## Tamer
+
+[Tamer](https://github.com/laserdisc-io/tamer) is a multi-functional Kafka connector for producing data based on [ZIO Kafka](https://github.com/zio/zio-kafka).
+
+### Introduction
+
+Tamer is a completely customizable source connector that produces to Kafka. It ships with preconfigured modules for SQL, cloud storage and REST API, but you can provide your own functions and Tamer will take care of the rest.
+
+### Installation
+
+Depending on the source you have at hand you can add the correct dependency in your `build.sbt`:
+
+```scala
+libraryDependencies += "io.laserdisc" %% "tamer-db"                % version
+libraryDependencies += "io.laserdisc" %% "tamer-oci-objectstorage" % version
+libraryDependencies += "io.laserdisc" %% "tamer-rest"              % version
+libraryDependencies += "io.laserdisc" %% "tamer-s3"                % version
+```
+
+### Example
+
+Let's say you have a inventory DB that's compatible with [Doobie](https://github.com/tpolecat/doobie), you can get all of your items with [just a few lines of code](example/src/main/scala/tamer/db/DatabaseSimple.scala):
+
+```scala
```

Comment:
It's great to see your work.
Adding mdoc modifiers (e.g. `mdoc scala:compile-only`) to your work would help us maintain documents over time.

Reply:
Wow, had no idea this was a thing

Reply:
You might find [this page](https://github.com/zio/zio/blob/series/2.x/docs/about/contributing-to-documentation.md) helpful, so it's worth a look.